packages/cli/src/module-resolver.ts

```diff
@@ -249,11 +289,5 @@ export function isCommonJs(
     return false;
   }
 
-  const { format } = resolve(
-    packageSpecifier,
-    pathToFileURL(parentUrl),
-    getFileSystemFromTypeScript(system),
-  );
-
-  return format === 'commonjs';
+  return getModuleType(packageSpecifier, system, parentUrl) === 'commonjs';
```

Comment:
What if this returns `directory`?

Reply:
It was only possible for relative imports, which is never the case for this call. I've simplified the code a bit though, so we never return `directory`, as it could be confusing in the future.